src/stats/discordStatsService.ts

```diff
@@ -0,0 +1,60 @@
+import { StatsService } from "./statsService";
+import { Client, TextChannel, Channel } from "discord.js";
+
+/**
+ * Uses the Discord API to implement the StatsService interface.
+ */
+export class DiscordStatsService implements StatsService {
+    private client: Client;
+
+    constructor(client: Client) {
+        this.client = client;
+    }
+
+    /**
+     * @returns the number of members in the server configured by the environment variables or 0 if there was an error
+     */
+    async getServerMemberCount(): Promise<number> {
+        const guildKey = process.env.DISCORD_SERVER_ID;
+        if (!guildKey) {
+            console.error(`ERROR: Couldn't get member count! Missing env. variable DISCORD_SERVER_ID. Please configure that value.`);
```

Comment:
This could be a normal string as there are no variables inside it

Reply:
You are right, I was just using them so that there are no conflicts when using the apostrophe (e.g. in "Couldn't"). I can change to simply using these: ".

Reply:
Ah ok. I usually escape the inner one, something like 

```typescript
console.error('User\'s list');

```

Reply:
Not a big deal or a blocker for the PR